machines/openIdBle/request.ts

```diff
@@ -109,6 +110,7 @@ export const requestMachine =
           target: '.inactive',
         },
         SCREEN_FOCUS: {
+          // eslint-disable-next-line sonarjs/no-duplicate-string
```

Comment:
Without eslint-disable, IDE keeps on showing error for duplicate code which are just states names that are supposed to be strings only. To avoid false error warning in IDE, I am adding this.